command.go

```diff
@@ -24,7 +24,7 @@ type commandReader struct {
 type Command struct {
 	client   *Client
 	shell    *Shell
-	ID       string
```

Comment:
I've updated all other ids to be lower case , the ID is exported but we don't need that attribute to be exported.